dds_cli/__main__.py

```diff
@@ -742,40 +768,62 @@ def create(
 
 
 ###################################################################################
-# SESSION ############################################################### SESSION #
+# AUTH ##################################################################### AUTH #
 ###################################################################################
-@dds_main.command()
+@dds_main.group()
+@click.pass_obj
+def auth(click_ctx):
+    """Manage the saved authentication token."""
+    pass
+
+
+@auth.command()
 @click.option(
     "--username",
     "-u",
     required=False,
     type=str,
     help="Your Data Delivery System username. Required unless the `--check` flag is used.",
 )
-@click.option(
-    "--check",
-    "-c",
-    required=False,
-    is_flag=True,
-    help="Instead of renewing the session, only check if the session is valid and report the token age.",
-)
 @click.pass_obj
-def session(_, username, check):
-    """Renew the access token stored in the '.dds_cli_token' file. Run this command before
+def login(click_ctx, username):
+    """Renew the authentication token stored in the '.dds_cli_token' file. Run this command before
     running the cli in a non interactive fashion as this enables the longest possible session time
     before a password needs to be entered again.
     """
+    no_prompt = click_ctx.get("NO_PROMPT", False)
+    if no_prompt:
+        LOG.warning("The --no-prompt flag is ignored for `dds auth login`")
     try:
-        with dds_cli.session.Session(username=username, check=check) as session:
-            if check:
-                session.check()
-            else:
-                # Session renewed in the init method.
-                LOG.info("[green] :white_check_mark: Session renewed![/green]")
+        with dds_cli.auth.Auth(username=username) as authenticator:
+            # Authentication token renewed in the init method.
+            LOG.info("[green] :white_check_mark: Authentication token renewed![/green]")
```

Comment:
We can skip the context manager here and just do `dds_cli.auth.Auth(username=username)` since the `authenticator` isn't used. Or alternatively `with dds_cli.auth.Auth(username=username):` ( without the "as authenticator" part)

Reply:
I went for the second option.